crates/router/src/core/payment_methods/transformers.rs

```diff
@@ -319,6 +320,7 @@ pub async fn mk_add_locker_request_hs(
     url.push_str("/cards/add");
     let mut request = services::Request::new(services::Method::Post, &url);
     request.add_header(headers::CONTENT_TYPE, "application/json".into());
+    request.add_header(headers::X_TENANT_ID, tenant_id.into());
```

Comment:
If it's not asking for too much, can we have our request ID sent to the locker in a header and have the locker include it in its logs, so that it helps debugging?

Can be taken up in a separate PR.